include/jwt-cpp/jwt.h

```diff
@@ -2873,11 +2875,243 @@ namespace jwt {
 		}
 	};
 
+	template<typename json_traits>
+	class jwk {
+		using basic_claim_t = basic_claim<json_traits>;
+		details::map_of_claims<json_traits> jwk_claims;
+
+	public:
+		JWT_CLAIM_EXPLICIT jwk(const typename json_traits::string_type& str) {
+			jwk_claims = details::map_of_claims<json_traits>::parse_claims(str);
+		}
```

Comment:
Oh code, it's been a few months I have not worked in C++ but I think we should use initializers

```suggestion
		JWT_CLAIM_EXPLICIT jwk(const typename json_traits::string_type& str) : 
					jwk_claims{details::map_of_claims<json_traits>::parse_claims(str)}
		{}
```